RecoTracker/SpecialSeedGenerators/python/CombinatorialSeedGeneratorForCosmics_cfi.py

```diff
@@ -9,31 +9,34 @@
 layerInfo = cms.PSet(
     MTIB = cms.PSet(
         TTRHBuilder = cms.string('WithTrackAngle'),
-            clusterChargeCut = cms.PSet(refToPSet_ = cms.string('SiStripClusterChargeCutNone')),
+        clusterChargeCut = cms.PSet(refToPSet_ = cms.string('SiStripClusterChargeCutNone')),
         rphiRecHits = cms.InputTag("siStripMatchedRecHits","rphiRecHit")
     ),
     TIB = cms.PSet(
         matchedRecHits = cms.InputTag("siStripMatchedRecHits","matchedRecHit"),
         TTRHBuilder = cms.string('WithTrackAngle'),
-            clusterChargeCut = cms.PSet(refToPSet_ = cms.string('SiStripClusterChargeCutNone'))
+        clusterChargeCut = cms.PSet(refToPSet_ = cms.string('SiStripClusterChargeCutNone'))
     ),
     MTOB = cms.PSet(
-        TTRHBuilder = cms.string('WithTrackAngle'),            clusterChargeCut = cms.PSet(refToPSet_ = cms.string('SiStripClusterChargeCutNone')),
+        TTRHBuilder = cms.string('WithTrackAngle'),
+        clusterChargeCut = cms.PSet(refToPSet_ = cms.string('SiStripClusterChargeCutNone')),
         rphiRecHits = cms.InputTag("siStripMatchedRecHits","rphiRecHit")
     ),
     TOB = cms.PSet(
         matchedRecHits = cms.InputTag("siStripMatchedRecHits","matchedRecHit"),
-        TTRHBuilder = cms.string('WithTrackAngle'),            clusterChargeCut = cms.PSet(refToPSet_ = cms.string('SiStripClusterChargeCutNone'))
+        TTRHBuilder = cms.string('WithTrackAngle'),
+        clusterChargeCut = cms.PSet(refToPSet_ = cms.string('SiStripClusterChargeCutNone'))
     ),
     TEC = cms.PSet(
         useSimpleRphiHitsCleaner = cms.bool(True),
         minRing = cms.int32(5),
         matchedRecHits = cms.InputTag("siStripMatchedRecHits","matchedRecHit"),
         useRingSlector = cms.bool(False),
-        TTRHBuilder = cms.string('WithTrackAngle'),            clusterChargeCut = cms.PSet(refToPSet_ = cms.string('SiStripClusterChargeCutNone')),
+        TTRHBuilder = cms.string('WithTrackAngle'),
+        clusterChargeCut = cms.PSet(refToPSet_ = cms.string('SiStripClusterChargeCutNone')),
         rphiRecHits = cms.InputTag("siStripMatchedRecHits","rphiRecHit"),
         maxRing = cms.int32(7)
-    ),
+  )
```

Comment:
I don't understand the Build errors, and the log does'n help me (@smuzaffar ?)

In any case, this parenthesis has to be brought back to the previous position to remain aligned. Maybe one can try to fix, and see if the Build error disappears...

Moreover @jeongeun could you please further:
- Add an empty line before [L56](https://github.com/cms-sw/cmssw/blob/f9855028cc992343cc71c798c77bfbf48d63819f/RecoTracker/SpecialSeedGenerators/python/CombinatorialSeedGeneratorForCosmics_cfi.py#L56), so that one can better notice where the EDProducer configuration actually starts (and perhaps also remove the empty line now uselessly inserted at [L58](https://github.com/cms-sw/cmssw/blob/f9855028cc992343cc71c798c77bfbf48d63819f/RecoTracker/SpecialSeedGenerators/python/CombinatorialSeedGeneratorForCosmics_cfi.py#L58)
- Align [L78-L89](https://github.com/cms-sw/cmssw/blob/f9855028cc992343cc71c798c77bfbf48d63819f/RecoTracker/SpecialSeedGenerators/python/CombinatorialSeedGeneratorForCosmics_cfi.py#L78-L89)

Reply:
> I don't understand the Build errors, and the log does'nt help me (@smuzaffar ?)
You can find comment from @smuzaffar here:  https://github.com/cms-sw/cmssw/pull/33863#issuecomment-859383526

Reply:
> > I don't understand the Build errors, and the log does'nt help me (@smuzaffar ?)
> 
> You can find comment from @smuzaffar here: [#33863 (comment)](https://github.com/cms-sw/cmssw/pull/33863#issuecomment-859383526)

Thank you @qliphy !